ui/src/copainput.tsx

```diff
@@ -63,6 +75,19 @@ export function CopaInput(props: any) {
       setSelectedImageHelperText("Image input can not have whitespace.")
     }
 
+    let seperateSplit = props.selectedImage.split(':');
+    let numColons = seperateSplit.length - 1;
+
+    if (numColons > 1) {
+      foundError = true;
+      setSelectedImageHelperText("Image input can only have one colon.");
+    } else {
```

Comment:
can we add another check case for if there is no colon? Either it should get latest as default or require the user input it

Reply:
currently it patches the latest as default, there could be a way to make this more obvious?

Reply:
small bug - if we do not add a tag, for example the image input is just "test" the error message says `Failed to patch test:` instead of `Failed to patch test:latest`